### PR TITLE
:recycle: cleaned up the lyrics dataset filtering and aligned equations to code

### PR DESCRIPTION
**Pull request by ruanvdmerwe**

### CODE REVIEW

File: practicals/large_language_models.ipynb [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "\n",
```
Comment (hereismari) [inline practicals/large_language_models.ipynb:16]: <p>Did some changes in the text / formatting: </p><p> </p><pre class="ql-syntax" spellcheck="false"># @title **Paths to follow:** What is your level of experience in the topics presented in this notebook? (Run Cell)
experience = "advanced" #@param ["beginner", "intermediate", "advanced"]
# TODO: update guidance or remove this cell.


sections_to_follow=""


if experience == "beginner":
  sections_to_follow = (
  """we recommend you to **not** attempt to do every coding task but
instead, skip through to every section and ensure you interact with the
LoRA finetuned LLM presented in the last section as well as with the pretrained
LLM to get a practical understanding of how these models behave"""
  )
elif experience == "intermediate":
  sections_to_follow = (
  """we recommend you go through every section in this notebook
and try the coding tasks tagged as beginner or intermediate.
If you get stuck on the code ask a tutor for help or move on to better use the time of the practical"""
  )
elif experience == "advanced":
  sections_to_follow = (
      """we recommend you go through every section
and try every coding task until you get it to work""")


print(f"Based on your experience, {sections_to_follow}.\nNote: this is just a guideline, feel free to explore the colab as you'd like if you feel confortable!")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2023/pull/49/discussion/#comment-1314246582'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

Comment (hereismari) [inline practicals/large_language_models.ipynb:16]: <pre class="ql-syntax" spellcheck="false">Line #3.    # TODO: update guidance or remove this cell.
</pre><p>Remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2023/pull/49/discussion/#comment-1314246583'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

Comment (hereismari) [inline practicals/large_language_models.ipynb:16]: <p>Thanks for the changes in the text here! looks great!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2023/pull/49/discussion/#comment-1314246584'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='23'/>

Comment (hereismari) [inline practicals/large_language_models.ipynb:16]: <pre class="ql-syntax" spellcheck="false">Line #10.    # TODO(ruan): verify if this is right.
</pre><p>Update/remove these comments</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2023/pull/49/discussion/#comment-1314246585'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='56'/>

Comment (hereismari) [inline practicals/large_language_models.ipynb:16]: <p>Add a note here that if people are not familiar with <code>einsum</code> to see the numpy documentation: https://numpy.org/doc/stable/reference/generated/numpy.einsum.html </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2023/pull/49/discussion/#comment-1314246586'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='136'/>

Comment (hereismari) [inline practicals/large_language_models.ipynb:16]: <p>nit: Can we keep the prompt I had here?  And add a note / task asking people to modify the prompt? Fine by me to keep this if you prefer ;)</p><p> </p><pre class="ql-syntax" spellcheck="false">formatted_artist_name = artist_name.replace('-', ' ').title()
prompt = f'This is a song by {formatted_artist_name}. It goes like this:\n\n' # @param
print(prompt)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2023/pull/49/discussion/#comment-1314246587'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='144'/>

Comment (ruanvdmerwe) [reply practicals/large_language_models.ipynb:16]: Changed

Comment (ruanvdmerwe) [reply practicals/large_language_models.ipynb:16]: <p>Added in </p>

Comment (ruanvdmerwe) [reply practicals/large_language_models.ipynb:16]: <p>Whoop forgot to change it back! Changing again</p><p> </p>

**Pull request merged**